react/features/authentication/functions.ts

```diff
@@ -27,7 +27,7 @@ export const isTokenAuthEnabled = (config: IConfig) =>
  * <tt>undefined</tt> if the pattern stored in config is not a string and the URL can not be
  * constructed.
  */
-export const getTokenAuthUrl = (config: IConfig, roomName: string) => {
+export const getTokenAuthUrl = (config: IConfig, roomName: string | undefined) => {
```

Comment:
This doesn't sound right. The token auth URL should only be gotten once we know the room. Please undo this and handle it at the calling site.